com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -627,21 +627,165 @@ half3 SampleLightmap(float2 lightmapUV, half3 normalWS)
+
+#if defined(UNITY_USE_NATIVE_HDR) || defined(UNITY_DOTS_INSTANCING_ENABLED)
+        irradiance += (1 - totalWeight) * encodedIrradiance.rbg;
+#else
+        irradiance += (1 - totalWeight) * DecodeHDREnvironment(encodedIrradiance, _GlossyEnvironmentCubeMap_HDR);
+#endif // UNITY_USE_NATIVE_HDR || UNITY_DOTS_INSTANCING_ENABLED
+    }
+
+    return irradiance;
+}
+
+half3 GlossyEnvironmentReflection(half3 reflectVector, float3 positionWS, half perceptualRoughness, half occlusion)
+{
+#if !defined(_ENVIRONMENTREFLECTIONS_OFF)
+    half3 irradiance = half3(0, 0, 0);
+
+#ifdef _REFLECTION_PROBE_BLENDING
+    irradiance = CalculateIrradianceFromReflectionProbes(reflectVector, positionWS, perceptualRoughness);
+#else
+#ifdef _REFLECTION_PROBE_BOX_PROJECTION
+    reflectVector = BoxProjectedCubemapDirection(reflectVector, positionWS, unity_SpecCube0_ProbePosition, unity_SpecCube0_BoxMin, unity_SpecCube0_BoxMax);
+#endif // _REFLECTION_PROBE_BOX_PROJECTION
+    half mip = PerceptualRoughnessToMipmapLevel(perceptualRoughness);
+    half4 encodedIrradiance = half4(SAMPLE_TEXTURECUBE_LOD(unity_SpecCube0, samplerunity_SpecCube0, reflectVector, mip));
+
+    //TODO:DOTS - we need to port probes to live in c# so we can manage this manually.
+#if defined(UNITY_USE_NATIVE_HDR) || defined(UNITY_DOTS_INSTANCING_ENABLED)
+    irradiance = encodedIrradiance.rgb;
+#else
+    irradiance = DecodeHDREnvironment(encodedIrradiance, unity_SpecCube0_HDR);
+#endif // UNITY_USE_NATIVE_HDR || UNITY_DOTS_INSTANCING_ENABLED
+#endif // _REFLECTION_PROBE_BLENDING
+
+    return irradiance * occlusion;
+#endif // _ENVIRONMENTREFLECTIONS_OFF
+
+    return _GlossyEnvironmentColor.rgb * occlusion;
+}
 half3 GlossyEnvironmentReflection(half3 reflectVector, half perceptualRoughness, half occlusion)
 {
 #if !defined(_ENVIRONMENTREFLECTIONS_OFF)
+    half3 irradiance = half3(0, 0, 0);
     half mip = PerceptualRoughnessToMipmapLevel(perceptualRoughness);
-    half4 encodedIrradiance = SAMPLE_TEXTURECUBE_LOD(unity_SpecCube0, samplerunity_SpecCube0, reflectVector, mip);
+    half4 encodedIrradiance = half4(SAMPLE_TEXTURECUBE_LOD(unity_SpecCube0, samplerunity_SpecCube0, reflectVector, mip));
 
-//TODO:DOTS - we need to port probes to live in c# so we can manage this manually.
+    //TODO:DOTS - we need to port probes to live in c# so we can manage this manually.
 #if defined(UNITY_USE_NATIVE_HDR) || defined(UNITY_DOTS_INSTANCING_ENABLED)
-    half3 irradiance = encodedIrradiance.rgb;
+    irradiance = encodedIrradiance.rgb;
 #else
-    half3 irradiance = DecodeHDREnvironment(encodedIrradiance, unity_SpecCube0_HDR);
-#endif
+    irradiance = DecodeHDREnvironment(encodedIrradiance, unity_SpecCube0_HDR);
+#endif // UNITY_USE_NATIVE_HDR || UNITY_DOTS_INSTANCING_ENABLED
 
     return irradiance * occlusion;
-#endif // GLOSSY_REFLECTIONS
+#endif // _ENVIRONMENTREFLECTIONS_OFF
 
```

Comment:
Same is true here but this preexisted your changes, but probably good to fix.

Reply:
resolved in 203240f